### DISCUSSION TITLE
Force relay with "/r" suffix does not work in command line

### DISCUSSION THREAD

**Discussion by davbe** [rustdesk/rustdesk#5837] [Q&A] [answered]
### Bug Description

When adding the /r suffix to an id, it forces the use of a relay server. This works fine when using the gui, but when using the command line (--connect 123456789/r) an error occurs (id does not exist).

### How to Reproduce

Run the command : rustdesk.exe --connect 123456789/r

### Expected Behavior

Should connect and use a relay.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

Windows 11

### Screenshots

NA

**Comment by davbe** [comment]
Also, a global option for forcing connections through relay server would be nice.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/commit/45dbea57f1c509875e4340254eeb252c657e15a1
